### ISSUE TITLE
Brand On is picking up any search for "on"

### ISSUE THREAD

**Issue by cklaassen** [bandsquad/google-trends-fetch#21]

**Comment by cklaassen** [comment]
One other Company name other than "On" is "Earth"

**Comment by cklaassen** [comment]
We need  to figure this out soon because it is going to mess up the results in the next-steps

**Comment by nathanparekh** [comment]
I'm guessing this is probably unavoidable... next-steps is just going to have to deal with it.

**Comment by cklaassen** [comment]
Do you think we could just skip searching those companies for now

**Comment by nathanparekh** [comment]
No, we can't just hardwire it. It's fine for now because we're only using the first 30 rows, and those arent until 90

**Comment by cklaassen** [comment]
pytrends.suggestions()
